ui/src/api/api.tsx

```diff
@@ -101,3 +102,41 @@ export const deleteFeature = async (qualifiedName: string) => {
     });
 };
 
+export const listUserRole = async () => {
+  let data:IUserRole[] = mockUserRole
+  return data
+};
+
+export const getUserRole = async (userName: string) => {
+  return await Axios
+  .get<IUserRole>(`${ API_ENDPOINT }/user/${userName}/userroles?code=${ token }`, {})
+  .then((response) => {
+    return response.data;
+  })
+}
+
+export const addUserRole = async (roleForm: RoleForm) => {
+  return await Axios
+  .post(`${ API_ENDPOINT }/user/${roleForm.userName}/userroles/new`, roleForm,
+      {
+        headers: { "Content-Type": "application/json;" },
+        params: {},
+      }).then((response) => {
+      return response;
+    }).catch((error) => {
+      return error.response;
+    });
+}
+
+export const deleteUserRole = async (roleForm: RoleForm) => {
+  return await Axios
+  .post(`${ API_ENDPOINT }/user/${roleForm.userName}/userroles/delete`, roleForm,
+      {
+        headers: { "Content-Type": "application/json;" },
+        params: {},
+      }).then((response) => {
+      return response;
+    }).catch((error) => {
+      return error.response;
+    });
+}
```

Comment:
it's not very good practice to bundle different things into one PR. Any special reason to do so? If not, let's do it in a separate PR. @Yuqing-cat

Reply:
Hi @hangfei I understand your concerns. Let me explain the reason here:
The lineage re-design PR contains some upgrades and restructure in frontend code structure. It's impossible to split all RBAC frontend content into a separate PR based on current main branch. In another word, the re-design PR is a hard dependency for all frontend changes. 

I strongly suggest us including it in this time to save engineer effort.
(Btw, the RBAC page is hided right now, users will not be able to access unless they know the redirect URL.)

Reply:
That works. Please update the PR description and the reasons to do so.